pkg/apis/apis/v1alpha1/types_apibinding.go

```diff
@@ -100,32 +100,31 @@ const (
 	ClaimRejected AcceptablePermissionClaimState = "Rejected"
 )
 
-// ExportReference describes a reference to an APIExport. Exactly one of the
+// BindingReference describes a reference to an APIExport. Exactly one of the
 // fields must be set.
-type ExportReference struct {
-	// workspace is a reference to an APIExport in the same organization. The creator
-	// of the APIBinding needs to have access to the APIExport with the verb `bind`
-	// in order to bind to it.
+type BindingReference struct {
+	// export is a reference to an APIExport by cluster name and export name.
+	// The creator of the APIBinding needs to have access to the APIExport with the
+	// verb `bind` in order to bind to it.
 	//
 	// +optional
-	Workspace *WorkspaceExportReference `json:"workspace,omitempty"`
+	Export *ExportBindingReference `json:"export,omitempty"`
 }
 
-// WorkspaceExportReference describes an API and backing implementation that are provided by an actor in the
-// specified Workspace.
-type WorkspaceExportReference struct {
-	// path is an absolute reference to a workspace, e.g. root:org:ws.
-	// If it is unset, the path of the APIBinding is used.
+// ExportBindingReference is a reference to an APIExport by cluster and name.
+type ExportBindingReference struct {
+	// path is a logical cluster path where the APIExport is defined.
+	// If the path is unset, the logical cluster of the APIBinding is used.
+	//
 	// +optional
-	// +kubebuilder:validation:Pattern:="^root(:[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$"
-	Path string `json:"path,omitempty"`
+	Path string `json:"cluster,omitempty"`
```

Comment:
Should the field be renamed to Cluster or the json reverted to path?

Reply:
nice finding. path is correct.